driver/src/driver/scheduler.rs

```diff
@@ -21,7 +21,7 @@ pub trait Scheduler {
     fn start(&mut self) -> !;
 }
 
-#[derive(Debug)]
+#[derive(Clone, Copy, Debug)]
```

Comment:
nit: I don't think this should be `Copy` and should just be `Clone`.

Reply:
I made it copy because
> Types whose values can be duplicated simply by copying bits.

Is that not correct?

Reply:
Hmm, good point.

Reply:
I guess this config is small enough that it doesn't really matter.